packages/core/src/contrib/Functions.ts

```diff
@@ -896,12 +924,10 @@ export const compDict = {
     numTicks: ad.Num,
     tickLength: ad.Num
   ): PathDataV<ad.Num> => {
-    if (typeof numTicks !== "number") {
-      throw Error("numTicks must be a constant");
-    }
     const path = new PathBuilder();
     // calculate scalar multipliers to determine the placement of each tick mark
-    const multipliers = tickPlacement(spacing, numTicks);
+    // TODO: why does the typecast work here?
+    const multipliers = tickPlacement(spacing, numTicks as number);
```

Comment:
This typecast should not work... what happens when you remove it?

Reply:
That's probably it. I should add it back (or really just do numOf)